commands/pigeon.rb

```diff
@@ -0,0 +1,51 @@
+class Pigeon < BaseCommand
+  require 'yaml'
+
+  def menu
+    @menu ||= YAML::load_file(File.join(__dir__, 'pigeon.yml')).symbolize_keys
+  end
+
+  def main
+    @main ||= menu[:mains].sample
+  end
+
+  def sample_sides
+    @sample_sides ||= menu[:sides].sample(rand(1..4))
+  end
+
+  def sides(number_of_sides = 4)
```

Comment:
Ah, yes, it is meant to be a param to

`@sample_sides ||= menu[:sides].sample(rand(1..4))` <- the 4